ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/ReportJobs.tsx

```diff
@@ -109,7 +109,10 @@ function ReportJobs({ reportId }: RunHistoryProps) {
                         >
                             <SelectOption value={runStates.PREPARING}>Preparing</SelectOption>
                             <SelectOption value={runStates.WAITING}>Waiting</SelectOption>
-                            <SelectOption value={runStates.SUCCESS}>Successful</SelectOption>
+                            <SelectOption value={runStates.GENERATED}>
+                                Download generated
+                            </SelectOption>
+                            <SelectOption value={runStates.DELIVERED}>Email delivered</SelectOption>
```

Comment:
@zhenpesky Let me know if the filter options text looks good to you

<img width="268" alt="Screenshot 2023-09-05 at 12 39 06 PM" src="https://github.com/stackrox/stackrox/assets/4805485/118cec8c-6a18-4ec1-9be1-2a75b4cdf69b">

Reply:
LGTM